internal/usergrant/repository/view/model/user_grant.go

```diff
@@ -57,7 +58,7 @@ type UserGrantView struct {
 func UserGrantToModel(grant *UserGrantView) *model.UserGrantView {
```

Comment:
do we need project owner only on this model?

Reply:
i think so as it's just for filtering data